src/guides.ts

```diff
@@ -0,0 +1,73 @@
+
+interface GuidesInterface {
+  document: Document
+  container: HTMLElement
+  classer?: (direction: string) => string
+  elements?: HTMLElement[]
```

Comment:
This doesn't necessarily need to be an `HTMLElement`, as it can be polymorphic i.e. accept anything with the right properties. We'll see how we can achieve this with Typescript.